oss-cloud-backend/serverless.yml

```diff
@@ -0,0 +1,18 @@
+service: oss-cloud-backend
+app: myapp
+org: aleksaleks123
```

Comment:
Had no access to this tenant, removing this line allowed starting the server (offline)

Reply:
Line 2 and line 3 are now removed.